### PR TITLE
[WFLY-19438] Add hook to disable Snappy compression

### PR DESCRIPTION
**Pull request by kabir**
https://issues.redhat.com/browse/WFLY-19438

### CODE REVIEW

File: microprofile/reactive-messaging-smallrye/kafka/src/main/java/org/wildfly/microprofile/reactive/messaging/config/kafka/ssl/context/KafkaReactiveMessagingSslConfigProcessor.java [hunk 1]
```diff
@@ -6,11 +6,16 @@
...
 import org.jboss.as.server.deployment.DeploymentPhaseContext;
+import org.jboss.as.server.deployment.DeploymentUnitProcessingException;
 import org.wildfly.microprofile.reactive.messaging.common.security.BaseReactiveMessagingSslConfigProcessor;
+import org.wildfly.security.manager.WildFlySecurityManager;
 
 import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
 
+import static org.wildfly.microprofile.reactive.messaging.config.kafka.ssl.context._private.MicroProfileReactiveMessagingKafkaLogger.LOGGER;
```
Comment (bstansberry) [inline microprofile/reactive-messaging-smallrye/kafka/src/main/java/org/wildfly/microprofile/reactive/messaging/config/kafka/ssl/context/KafkaReactiveMessagingSslConfigProcessor.java:17]: Order of import sections is reversed from what it should be.

File: microprofile/reactive-messaging-smallrye/kafka/src/main/java/org/wildfly/microprofile/reactive/messaging/config/kafka/ssl/context/_private/MicroProfileReactiveMessagingKafkaLogger.java [hunk 2]
```diff
@@ -33,4 +34,7 @@ public interface MicroProfileReactiveMessagingKafkaLogger extends BasicLogger {
     @Message(id = 2, value = "Could not find an Elytron client-ssl-context called: %s")
     IllegalStateException noElytronClientSSLContext(String ctx);
 
+    @Message(id = 3, value = "Snappy compression is not supported when running on Windows. The MicroProfile Config " +
```
Comment (bstansberry) [inline microprofile/reactive-messaging-smallrye/kafka/src/main/java/org/wildfly/microprofile/reactive/messaging/config/kafka/ssl/context/_private/MicroProfileReactiveMessagingKafkaLogger.java:37]: s/Windows/Windows or OSX/g

**Review by bstansberry — CHANGES_REQUESTED** [review]
Minor point about the error message.

It looks like your IDE import settings need a tweak. ;)

**Comment by kabir** [comment]
@bstansberry Updated

**Review by bstansberry — APPROVED** [review]

**Pull request merged**